sympy/matrices/matrices.py

```diff
@@ -3963,11 +3963,14 @@ def row_join(self, rhs):
         row
         col_join
         """
+        from sympy.matrices import MutableMatrix
+        #Allows you to build a matrix even if it is null matrix
+        if (self.rows <= 0 and self.cols <= 0):
```

Comment:
here and below: `if not self:`. That will work because `bool(zeros(42))==True`. The only matrices that have bool()->False are those with a 0 dimension.